backend/research/views.py

```diff
@@ -19,12 +20,12 @@ class ProjectView(viewsets.ModelViewSet):
 
     def get_queryset(self):
         # TODO: HOD should be able to see all
-        return Project.objects.filter(author__id=self.request.user)
+        return Project.objects.filter(authors__email=self.request.user.email)
```

Comment:
no need to make explicit email field. If we specify `authors=user`, it will internally get converted to the pk